docs/wiki/development/coding-guidelines.md

```diff
@@ -178,6 +178,10 @@ Every function should have a header of the following format as the start of thei
  * Arguments:
  * 0: The first argument <STRING>
  * 1: The second argument <OBJECT>
+ * 2: Multiple input types <STRING|ARRAY|CODE>
```

Comment:
this doesn't look consistent with https://github.com/acemod/ACE3/pull/5253/files#diff-24615c452e26c6f9032ec7e9bc660c61R156 (`\<([\s\w]+?)\>`)

Reply:
which could be changed